src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  margin: 0;
```

Comment:
It is better to use (*) selector only for box-sizing

Reply:
Done